src/validation.cpp

```diff
@@ -1506,61 +1506,68 @@ bool DisconnectBlock(const CBlock &block, CValidationState &state,
 
     if (pfClean) *pfClean = false;
```

Comment:
can remove as pfClean is passed through to ApplyBlockUndo() and is initialized there if needed